web/client/api/ScaleBox.js

```diff
@@ -0,0 +1,152 @@
+/**
+ * Copyright 2015, Marko Polovina
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/**
+ * [ScaleBox component]
+ * @param {[Boolean]} showBar   [if you want to display scalebar]
+ * @param {[Boolean]} showCombo [if you want to display scalecombo]
+ * @param {[String]} units [units as string, eg. 'm', 'km', 'mi']
+ * @param {[Object]} options   [component options]
+ */
+var ScaleBox = function(map, showBar, showCombo, units, options) {
```

Comment:
React components should be self contained, api utilities should be used for general functionality, not for UI purposes